include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -154,21 +153,24 @@ concept bool aligned_sequence_concept =
 //!\endcond
 
 // -----------------------------------------------------------------------------
-// Helper functions for stl container modelling the aligned_sequence_concept
+// Helper functions for stl containers to model the aligned_sequence_concept
```

Comment:
Maybe "// Functions that make sequence containers model the aligned_sequence_concept"?